docs/requirements.txt

```diff
@@ -8,6 +8,7 @@ python-qpid-proton
 psycopg2-binary
 recommonmark
 requests
+ruamel.yaml
```

Comment:
Based on the recommendation here (https://yaml.readthedocs.io/en/latest/basicuse.html) Should this be pinned to a version <0.15?

Reply:
The version less than 0.15 is 3 years old. Also, the operator-manifests library uses 0.16.x which does most of the modifications. Also, when installing that old version, I get a traceback on Python 3.7. Lastly, the version in Fedora 31 is 0.16.5, so I'll assume that it's safe to use.